plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryConnectorTest.java

```diff
@@ -770,6 +770,9 @@ public void testNativeQuerySelectFromNation()
         assertQuery(
                 "SELECT * FROM TABLE(bigquery.system.query(query => 'SELECT name FROM tpch.nation WHERE nationkey = 0'))",
                 "VALUES 'ALGERIA'");
+        assertQuery(
```

Comment:
This only verifies correctness - not pushdown as far as I can tell.

Reply:
We need to improve `BigQueryMetadata#applyFilter` to let `isFullyPushedDown` true. Let me handle in a follow-up PR.